package.json

```diff
@@ -17,7 +17,6 @@
     "browserify-preload-component": "browserify -o lib/src/renderer/_preload-component.js -x electron --insert-global-vars=__filename,__dirname lib/src/renderer/preload-component.js",
     "compile": "run-s lint compile:*",
     "compile:project": "gulp build",
-    "compile:spec": "tsc -p tsconfig.spec.json",
```

Comment:
We need this for unit tests right?

Reply:
As per my understanding, this tsconfig file is specific to Spectron which is not used anymore